src/gdpr.erl

```diff
@@ -1,11 +1,16 @@
 -module(gdpr).
 
 -export_type(
-    [data_group/0,
+    [username/0,
+    domain/0,
```

Comment:
why do we need a new type if in fact we want it to be 1-2-1 compatible with mongooseim:domain_name()

Reply:
idk, I am just exporting types because they are used in other modules, I am not the original author of the types.